validator/src/main.rs

```diff
@@ -1754,8 +1754,10 @@ pub fn main() {
         BlockProductionMethod
     )
     .unwrap_or_default();
-    validator_config.enable_block_production_forwarding =
-        matches.is_present("enable_block_production_forwarding");
+    validator_config.enable_block_production_forwarding = staked_nodes_overrides_path
+        .as_ref()
+        .map(|_| matches.is_present("disable_block_production_forwarding"))
```

Comment:
`.map(|_| !matches.is_present("disable_block_production_forwarding"))`
( `!` is missed )

Reply:
that's what i get for trying to get it before i ran out the door. Fixed: c76b85354e